lib/mkvm/version.rb

```diff
@@ -1,3 +1,3 @@
 module MKVM
-	VERSION = '3.1.2'
+	VERSION = '3.2.0'
```

Comment:
Removing Satellite Support seems like a major version rev to me as it's a breaking change for anyone using Satellite.

Reply:
That's valid. I don't think anyone's really using any of this code as-is. We mostly put it out there as a reference for people to use (specifically the rbvmomi stuff). That being said, maybe it's worth just leaving the satellite stuff there as a reference as well

Reply:
I went ahead & re-added the satellite support for rmvm. We're not using it anymore but it could be useful for someone else as example code.